voluncheer/requirements.txt

```diff
@@ -27,4 +27,4 @@ six==1.16.0
 sqlparse==0.4.3
 urllib3==1.26.14
 verify-email==2.4.3
-Pillow==9.3.0
+Pillow==9.3.0
```

Comment:
I don't think `Pillow` is used anymore. This can be removed. It is from an old commit.

Reply:
I was wrong about this, leave it as is.